dataland-frontend/src/assets/scss/components/base.scss

```diff
@@ -236,3 +236,7 @@ body {
 .p-disabled, .p-component:disabled {
   opacity: .5;
 }
+
+.d-cursor-pointer {
```

Comment:
Does someone know what this `d-` prefix is for? Dataland? Like `p-` is for primevue (I assumed)?

Reply:
For Dataland was at least my understanding

Reply:
We should make people more aware of this because I, for example, never used this prefix before.